### PR TITLE
Update storage-ng warnings for o3

### PR DESCRIPTION
**Pull request by mloviska**
- Related ticket: [[openSUSE][functional][y][yast] Check warning for too small PReP (ppc), EFI (aarch64), Bios Boot (x86_64+GPT) or root partition / on openSUSE](https://progress.opensuse.org/issues/40940)
- Needles: [o3](https://github.com/os-autoinst/os-autoinst-needles-opensuse/pull/440)
- Verification runs:
1. [leap15.1](http://dhcp128.suse.cz/tests/6432#step/partitioning_warnings/1)
2. [sle15sp1](http://dhcp128.suse.cz/tests/6440#step/partitioning_warnings/1)

[Fix sle12sp4](http://dhcp128.suse.cz/tests/6485#step/partitioning_warnings/1) for reported issue on [ARM](https://openqa.suse.de/tests/2094111#step/partitioning_filesystem/1)

TW is currently blocked by [poo#41357](https://progress.opensuse.org/issues/41357)

### CODE REVIEW

File: tests/installation/partitioning_warnings.pm [hunk 1]
```diff
@@ -100,7 +100,7 @@ sub run {
     }
     # create small enough partition (11GB) to get warning for enabled snapshots
     # on storage-ng snaphots are disabled as per proposal when partition is to small, so enable to check the warning
-    addpart(role => 'OS', size => 11000, format => 'btrfs', enable_snapshots => is_storage_ng);
+    addpart(role => 'OS', size => (is_opensuse) ? 9000 : 11000, format => 'btrfs', enable_snapshots => is_storage_ng);
```
Comment (okurz) [inline tests/installation/partitioning_warnings.pm]: Could you explain me why openSUSE is different here?

**Comment by okurz** [comment]
@dzedro you might be interested in this PR as a test module maintainer

File: tests/installation/partitioning_warnings.pm (hunk 1 above)
Comment (mloviska) [reply tests/installation/partitioning_warnings.pm]: Thank you for updating the comments, I have miss this part. I am not really aware how the algorithm works so I have been following the warnings [leap15.1 - 10GB](https://openqa.opensuse.org/tests/764759#step/partitioning_warnings/37) and for [sle15sp1 - 12 GB](https://openqa.suse.de/tests/2095938#step/partitioning_warnings/37).
Also there is a different threshold for small root [leap15.1 - 5GB](https://openqa.opensuse.org/tests/764759#step/partitioning_warnings/148) and [sle15sp1 - 3GB](https://openqa.suse.de/tests/2095938#step/partitioning_warnings/148)

Comment (okurz) [reply tests/installation/partitioning_warnings.pm]: @DimStar77 @lnussel @fcrozat is the difference in the root partition limit under which no snapshots are proposed among TW/Leap/SLE on purpose? I guess these are settings in control.xml files?

**Comment by dzedro** [comment]
LGTM

File: tests/installation/partitioning_warnings.pm (hunk 1 above)
Comment (fcrozat) [reply tests/installation/partitioning_warnings.pm]: > @DimStar77 @lnussel @fcrozat is the difference in the root partition limit under which no snapshots are proposed among TW/Leap/SLE on purpose? I guess these are settings in control.xml files?
Yes, it is part of each distro control.xml or even the system roles, these days.

**Pull request merged**